frontend/src/Settings/UI/UISettings.js

```diff
@@ -111,9 +116,21 @@ class UISettings extends Component {
                   </FormGroup>
                 </FieldSet>
 
-                <FieldSet
-                  legend={translate('Dates')}
-                >
+                <FieldSet legend="Movies">
+                  <FormGroup>
+                    <FormLabel>Runtime Format</FormLabel>
```

Comment:
Add translation key `SettingsRuntimeFormat` and changed this to translated

Reply:
Added